app/components/Home.tsx

```diff
@@ -12,6 +12,7 @@ import CompanyValues from "./CompanyValues";
 import Machines from "@/app/components/Machines";
 import RequestQuote from "./RequestQuote";
 import Location from "./Location";
+import MachineCarousel from "./MachineCarousel";
```

Comment:
Sorry, missed this import when I was resolving merge conflicts
```suggestion
```

Reply:
Okay, I was going to point out that you could remove it